### PR TITLE
Add hard negative mining eval 

### PR DESCRIPTION
**Pull request by bicycleman15**

### CODE REVIEW

File: trainAdamWC1.py [hunk 1]
```diff
@@ -249,9 +249,11 @@ def train_eval(model,val_data,batch_size_val,epoch_no):
...
             loss = cfg['loc_weight'] * loss_l + loss_c + loss_landm
             loss_val += loss.item()
-
-    # loss_val = loss_val / (batch_size * len(val_data))  # get average loss per image
-    print('Validation loss for Epoch {} : {}'.format(epoch_no,loss_val))
+    if not is_ohem:
+        # loss_val = loss_val / (batch_size * len(val_data))  # get average loss per image
+        print('Validation loss for Epoch {} : {}'.format(epoch_no,loss_val))
+    else:
+        print('Ohem loss for Epoch {} : {}'.format(epoch_no,loss_val))
```
Comment (neelabh17) [inline trainAdamWC1.py:256]: Did you add the HNM images in the data folder with older training images??

Comment (bicycleman15) [reply trainAdamWC1.py:256]: No not yet.

Comment (bicycleman15) [reply trainAdamWC1.py:256]: I mean you just need add those new images in `train` folder as well as add those images to a new folder named `ohem`. That's it. It'll work

Comment (neelabh17) [reply trainAdamWC1.py:256]: So right now the train loss does not contain loss coming from `hard negative examples`?

Comment (bicycleman15) [reply trainAdamWC1.py:256]: `train_loss` is from both `train` and `ohem` images(becuase we will add `ohem` images with the `train` images), as it should be. But `ohem_loss` is from `ohem` images only because we want to track those.

**Review by neelabh17 — APPROVED** [review]
You have done a good job.
